org.zotero.Zotero.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "org.zotero.Zotero",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "zotero",
+    "rename-desktop-file": "zotero.desktop",
+    "rename-icon": "zotero",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
this is an electron app right? that doesn't currently support wayland.

Reply:
It's based on XUL (old firefox engine).

Reply:
ah, ok - even then - I know that upstream firefox hasn't enabled Wayland for people by default yet. Have you tested this on a wayland system?

Reply:
I'm on a Wayland, but it might be safer to force XWayland.